runtime/shared_common/shrinit.cpp

```diff
@@ -3091,6 +3123,16 @@ j9shr_init(J9JavaVM *vm, UDATA loadFlags, UDATA* nonfatal)
 
 	Trc_SHR_INIT_j9shr_init_Entry(currentThread);
 
+	/* noSemaphoreCheck and checkClasspathTimestamp shouldn't coexist no matter what order
+	 * they are specified on the command line.
+	 */
+	if (J9_ARE_NO_BITS_SET(runtimeFlags, J9SHR_RUNTIMEFLAG_ENABLE_TIMESTAMP_CHECKS)
+	&& J9_ARE_ALL_BITS_SET(runtimeFlags, J9SHR_RUNTIMEFLAG_ENABLE_CLASSPATH_TIMESTAMP_CHECK)
+	) {
+		SHRINIT_ERR_TRACE2(1, J9NLS_SHRC_SHRINIT_INCOMPATIBLE_OPTION, OPTION_NO_TIMESTAMP_CHECKS, OPTION_CLASSPATH_TIMESTAMP_CHECK);
```

Comment:
I think this can just be warning. If this condition occurs then J9SHR_RUNTIMEFLAG_ENABLE_CLASSPATH_TIMESTAMP_CHECK will just be ignored.

Reply:
Already changed the code to ignore J9SHR_RUNTIMEFLAG_ENABLE_CLASSPATH_TIMESTAMP_CHECK after trigger a warning.